docs/index.md

```diff
@@ -116,9 +116,12 @@ This allows the provider to use the SSH agent configured by the user, and to sup
 You can find more details on the SSH Agent [here](https://www.digitalocean.com/community/tutorials/ssh-essentials-working-with-ssh-servers-clients-and-keys#adding-your-ssh-keys-to-an-ssh-agent-to-avoid-typing-the-passphrase).
 The SSH agent authentication takes precedence over the `private_key` and `password` authentication.
 
+> [!NOTE]
+> By default on Windows, the provider will assume the SSH agent is at `\\.\pipe\openssh-ssh-agent`.
```

Comment:
Terraform docs uses a slightly different notation for callouts, info is `->`, warning is `~>`, critical is `!>` 
```suggestion
-> By default on Windows, the provider will assume the SSH agent is at `\\.\pipe\openssh-ssh-agent`.
```